### PR TITLE
Use question id instead of question number when moderating students #5210

### LINKED ISSUE #5210: Use question id instead of question number when moderating students
Use feedbackQuestionId instead of questionNumber to identify which question is getting moderated.

### PR DESCRIPTION
**Pull request by kennho**
Fixes #5210

### CODE REVIEW

**Comment by kennho** [comment]
@kanghj is this correct?

File: src/main/java/teammates/ui/template/InstructorFeedbackResultsModerationButton.java [hunk 1]
```diff
@@ -6,6 +6,7 @@
     
     private boolean isDisabled;
     private String buttonText;
+    private String questionId;
     private int questionNumber;
```
Comment (kanghj) [inline src/main/java/teammates/ui/template/InstructorFeedbackResultsModerationButton.java]: @kennho, will it be possible to remove this questionNumber field entirely?

Comment (kennho) [inline src/main/java/teammates/ui/template/InstructorFeedbackResultsModerationButton.java]: okay

**Comment by kennho** [comment]
Updated with suggested changes @kanghj running tests to update the pages now

**Comment by kennho** [comment]
updated with tests @kanghj

File: src/main/java/teammates/ui/controller/InstructorEditStudentFeedbackPageAction.java [hunk 2]
```diff
@@ -73,8 +73,8 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         data.setHeaderHidden(true);
         data.setStudentToViewPageAs(studentUnderModeration);
         
-        if (moderatedQuestionNumber != null) {
-          data.setModeratedQuestion(moderatedQuestionNumber);
+        if (moderatedQuestionId != null) {
+          data.setModeratedQuestionId(moderatedQuestionId);
```
Comment (kanghj) [inline src/main/java/teammates/ui/controller/InstructorEditStudentFeedbackPageAction.java]: Indentation here

**Comment by kanghj** [comment]
No other comments from me. @damithc for your review

**Pull request merged**